packages/astro/src/core/build/plugins/plugin-content.ts

```diff
@@ -308,8 +425,15 @@ export function pluginContent(
 	opts: StaticBuildOptions,
 	internals: BuildInternals
 ): AstroBuildPlugin {
-	const cachedChunks = new URL('./chunks/', opts.settings.config.cacheDir);
-	const distChunks = new URL('./chunks/', opts.settings.config.outDir);
+	const { cacheDir, outDir } = opts.settings.config;
+
+	const chunksFolder = './chunks/';
```

Comment:
Where is this `chunks` folder coming from? Is it a rollup thing? If so, I think we should compute some setting because it can change if the build changes

Reply:
This is the same hardcoded value in `static-build.ts`. Should it be moved somewhere as a constant and used in both places?

Reply:
Maybe it could be useful, so we can inline some comment that explains that's that folder for

Reply:
Moved it to a constant.